satellite/orders/endpoint.go

```diff
@@ -44,6 +44,9 @@ type DB interface {
 	// UpdateStoragenodeBandwidthSettle updates 'settled' bandwidth for given storage node
 	UpdateStoragenodeBandwidthSettle(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64) error
 
+	// UpdateStoragenodeBandwidthSettleWithCustomDate updates 'settled' bandwidth for given storage node for the given intervalStart time
+	UpdateStoragenodeBandwidthSettleWithCustomDate(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
```

Comment:
This method makes me nervous.  There is no way of knowing (without reading all the code) that this method is supposed to be used in testing.  Im worried that someone could accidentally use it in production code instead of the `UpdateStoragenodeBandwidthSettle` which _should_ be used.  

For the test that uses this method, I'm curious if there is a way we can mock out the time package when creating the `storagenode_bandwidth_rollups` instead of adding this method. Does that make sense?

Reply:
I definitely see the issue, and Cam pointed that out too. I'm not sure how to fix it.

Reply:
As an example, I'm reading this about mocking time package: https://stackoverflow.com/questions/18970265/is-there-an-easy-way-to-stub-out-time-now-globally-in-golang-during-test

Reply:
maybe something like this? https://www.reddit.com/r/golang/comments/a8yx3x/mocking_system_time_in_tests/

Reply:
oops I didn't see cam's comment til now, didn't refresh.